tads/CMakeLists.txt

```diff
@@ -47,11 +47,22 @@ if(WITH_TADS)
   list(APPEND srcs tads3/vmconnom.cpp)
 
   set(macros GARGOYLE VMGLOB_STRUCT GLK_UNICODE GLK_TIMERS TC_TARGET_T3 RUNTIME)
+  if(LINUX)
+    list(APPEND macros GLK_ANSI_ONLY)
+  endif()
```

Comment:
What is the purpose of defining this on Linux?  From what I can tell this will provide a text-only prompt instead of using Glk, so (for example) if you try restoring, TADS will just print this:

```Restore game from file > ```

rather than popping up a dialog box (or using whatever the Glk implementation deems appropriate).